app/models/box_request.rb

```diff
@@ -24,12 +24,14 @@ class BoxRequest < ApplicationRecord
   scope :assembled, ->(){ joins(:box).where("boxes.aasm_state = ?", "assembled") }
   scope :shipping_in_progress, ->(){ joins(:box).where("boxes.aasm_state = ?", "shipping_in_progress") }
   scope :shipped, ->(){ joins(:box).where("boxes.aasm_state = ?", "shipped") }
+  scope :researched, ->(){ joins(:box).where("boxes.aasm_state = ?", "researched") }
 
   aasm do
 
     state :requested, :initial => true
     state :review_in_progress
     state :reviewed
+    state :researched
```

Comment:
hey there! `researched` is not a state on `box_request`, it's a state on `box`. the scope above joins(:box), so that's why it's ok that the state isn't in `box_request.rb`

Reply:
🤦‍♂ that makes way more sense. I'll update the PR. thanks!